site/public/apps/dermatotarget-atlas/index.html

```
@@ -0,0 +1,67 @@
+
+      <div class="sidebar-foot">
+        <span class="hyp-badge" title="All outputs are prioritization hypotheses, not clinical guidance.">
+          Hypothesis-generating
+        </span>
+        <span class="run-stamp" id="run-stamp"></span>
+      </div>
+    </aside>
+
+    <div class="content">
+      <header class="topbar" role="toolbar" aria-label="Page tools">
+        <button class="menu-toggle" id="menu-toggle" aria-label="Toggle navigation" aria-expanded="false">Menu</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)